packages/core/core/src/Parcel.js

```diff
@@ -104,6 +119,20 @@ export default class Parcel {
     );
 
     this.runPackage = this.farm.mkhandle('runPackage');
+
+    if (this.options.serve) {
```

Comment:
Not sure if this should be in core, seems like it would be better fit in cli as people who use the API will probably not want the server I guess

Reply:
Perhaps we could even make the server a reporter of some sort?